src/content/docs/release-notes/docs-release-notes/docs-4-19-2024.mdx

```diff
@@ -0,0 +1,42 @@
+---
+subject: Docs
+releaseDate: '2024-04-19'
+version: '4-19-2024'
+---
+
+<Callout variant="important">
+We're trying out release notes to let you know what's changing in our docs each week. This is an experiment, so [let us know via GitHub issue](https://github.com/newrelic/docs-website/issues/new/choose) if you find them helpful!
+</Callout>
+
+### Major changes
+
+* Continued rewrite of on-host integration docs to make it easier to install and ensure the docs are accurate and up to date. Up this week: [Ray integration](/docs/infrastructure/host-integrations/host-integrations-list/ray-integration) and [NGINX]().
+* Removed docs related to a deprecated anomaly detection feature.
+* Updated [Ruby agent configuration](/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration) for version 9.9.0, including `thread_local_tracer_state` and serverless mode.
+* Reorganized our OpenTelemetry docs and moved the category to the top-level nav.
+* Removed section on setting up FreshService alerts notification integration, because we no longer support it.
```

Comment:
```suggestion
* Removed section on setting up Freshservice alerts notification integration, because we no longer support it.
```

Reply:
Not entirely sure about this, but it seems right - https://www.freshworks.com/freshservice/